browser-ui/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>wasm-python terminal</title>
+    <link rel="stylesheet" href="https://unpkg.com/xterm@4.16.0/css/xterm.css" crossorigin/>
+    <link rel="stylesheet" href="/styles.css"/>
```

Comment:
Make import relative so we can deploy the shell in a sub directory on Ethan's GH page.
```suggestion
    <link rel="stylesheet" href="./styles.css"/>
```